src/context_store/ingestion/task_registry.py

```
@@ -0,0 +1,92 @@
+            )
+
+    async def cancel_all(self, timeout: float = 5.0) -> None:
+        """Cancel all running tasks with timeout. Called during graceful shutdown."""
+        if not self._tasks:
+            return
+
+        tasks = list(self._tasks)
+        for task in tasks:
+            task.cancel()
+
+        # タスクの完了を待機（タイムアウト付き）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)